BC.SaasHelper.ps1

```diff
@@ -1,10 +1,12 @@
 # SaaS specific functions
+. (Join-Path $PSScriptRoot "SaaSHelperFunctions.ps1")
```

Comment:
This one also should not be here - the SaaSHelperFunctions should be included elsewhere - all the other functions use it as well